### DISCUSSION TITLE
Welcome to Langflow Discussions!

### DISCUSSION THREAD

**Discussion by carlosrcoelho** [langflow-ai/langflow#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * To be as welcoming and recognize your other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by websmyths** [comment]
Hi, I'm a web developer, but relatively new to AI, language models, and Python. I have played around with Langchain and OpenAI APIs. Stumbled across this and it is very cool. Played around briefly with the demo. I have a couple questions, I will start a topic on them. Thx.

**Comment by shaojiezhao** [comment]
hi all, Modify the source code then with which Docker file to build a new image,there are so many Docker file.

**Reply by sidoboy** [reply]
Hi

**Comment by PerttyZhan** [comment]
I want to deploy lang-flow to my local，Should I use the Dockerfile in deploy or the Dockerfile in the root directory?
